terraform/domains/environment_domains/config/dev.tfvars.json

```diff
@@ -1,6 +1,41 @@
 {
-  "domains": ["dev"],
-  "environment_short": "dv",
-  "environment_tag": "dev",
-  "origin_hostname": "trs-dev-api.test.teacherservices.cloud"
+  "hosted_zone": {
+    "teacher-qualifications-api.education.gov.uk": {
+      "front_door_name": "s189p01-trsdomains-fd",
+      "resource_group_name": "s189p01-trsdomains-rg",
+      "domains": [
+        "dev"
+      ],
+      "cached_paths": [
```

Comment:
I'm not sure there is a /assets path to cache. @gunndabad please confirm if you'd like caching

Reply:
There's nothing at `assets`; we don't need any caching.